signac_dashboard/templates/plot_viewer/js/plot_viewer.js

```diff
@@ -0,0 +1,12 @@
+function draw_plot(element) {
+    data = JSON.parse(element.getAttribute("data-plotly-data"));
```

Comment:
Maybe the card HTML can contain a REST endpoint, and fetch the data with a second call? That way we don't have to generate and include all the raw data in the HTML page. This should help a lot with page responsiveness. You can see examples of REST calls in other modules:
- https://github.com/glotzerlab/signac-dashboard/blob/master/signac_dashboard/templates/notes/js/notes.js
- https://github.com/glotzerlab/signac-dashboard/blob/master/signac_dashboard/templates/document_editor/js/document_editor.js

If you need help with this, let me know.

Reply:
I have implemented this in 226111ef967cd294ebace776d82d0bca2b094d58. The page loading is way more responsive now.

Still, I'm a total newbie with Flask, so a thorough review is needed.

My main concerns are:

- I have had to create a module identifier so that a different endpoint is created for each instance of the `PlotlyView` module. I decided to hash the `id(self)` which should work, but has the disadvantage that each time the dashboard is launched the URLs change. I don't think this is a huge problem, but I feel like there is a better way to do this.
- For each added instance of the module, the `register` method is called. However, the assets URLs should only be added once. If you try to do it multiple times, the second time around the module is not registered. I have solved this with a class attribute `PlotlyViewer._assets_url_registered` that avoids this duplication. Still, I feel like this solution is  somewhat hacky.

Reply:
@javierbg 
Flask Blueprints are the way to go. I started experimenting with them in #182 to allow multiple Notes modules to be added.